### PR TITLE
added support for Vivaldi web browser by modifying chrome functions

### PR DESCRIPTION
**Pull request by TripsJ**
I Basically copied the chrome approach for the vivaldi web browser vivaldi and added the corresponding choice

### CODE REVIEW

File: grab-x-link.el [hunk 1]
```diff
@@ -1,45 +1,5 @@
-;;; grab-x-link.el --- Grab links from X11 apps and insert into Emacs  -*- lexical-binding: t; -*-
```
Comment (xuchunyang) [inline grab-x-link.el]: The whole header section is required and should not be deleted.

Comment (xuchunyang) [reply grab-x-link.el]: other changes look good to me.

Comment (TripsJ) [reply grab-x-link.el]: I kinda did that by accident. I'll put it back in tomorrow and push it up,

**Comment by TripsJ** [comment]
Header back in,  Changelog updated

**Pull request merged**